### PR TITLE
Issue #1: Create an entry

### PR DESCRIPTION
**Pull request by jamesdae**
https://user-images.githubusercontent.com/110082070/199412811-617f7d5b-76d2-4746-9d8f-5aae4f0bf816.mov

### CODE REVIEW

File: js/main.js [hunk 1]
```diff
@@ -0,0 +1,25 @@
...
+  newObject.titleinput = $titleInput.value;
+  newObject.photoinput = document.forms[0].elements.photoinput.value;
+  newObject.notesinput = document.forms[0].elements.notesinput.value;
+  newObject.entryId = data.nextEntryId;
+  data.nextEntryId += 1;
+  data.entries.unshift(newObject);
+  $image.removeAttribute('src');
+  $image.setAttribute('src', 'images/placeholder-image-square.jpg');
+  var formJSON = JSON.stringify(data);
+  localStorage.setItem('javascript-local-storage', formJSON);
```
Comment (BrettAlbright) [inline js/main.js]: this should be done in the `data.js` file.

**Review by Zhiwutian — CHANGES_REQUESTED** [review]
Hey James, looking great so far! The only update for this is to move the local storage functionality into the `data.js` file, also, the local storage functionality should match the functionality in the javascript local storage exercise, where you store the whole object in local storage when the page is refreshed, and the data is reloaded when the app starts. That is the only time you should be accessing local storage, not during the normal running of the application. 

Reach out if you have questions!

**Comment by jamesdae** [comment]
https://user-images.githubusercontent.com/110082070/199560797-03381750-000b-4094-9ef5-96081c592717.mov

https://user-images.githubusercontent.com/110082070/199560814-5d89c324-eb77-440a-be6b-484cac6d2903.mov

https://user-images.githubusercontent.com/110082070/199560846-4f8ddbac-c418-4cbb-8e85-020de65e2e20.mov

**Review by Zhiwutian — APPROVED** [review]
👍🏼

**Pull request merged**